docs/src/index.md

```diff
@@ -2,7 +2,7 @@
 
 *Lightweight robust covariance estimation in Julia.*
 
-A package for estimating covariance matrices.
+A package for robustly estimating covariance matrices of real-valued data.

```

Comment:
I don't think it's necessary to write that the package is for real-valued data only. It currently is now but I'd be happy to accept new methods that work for non-real valued data.

Reply:
I think that's great but for now it's accurate to specify that we only deal with real-values; maybe leave this there and once we've started seriously tackling complex-valued stuff we remove it?

Reply:
OK, we can leave this for now.